tools/clang/test/CodeGenSPIRV/type.texture.hlsl

```diff
@@ -72,6 +73,19 @@ Texture1D<S>      sTex;
 // CHECK: %tTex = OpVariable %_ptr_UniformConstant_type_2d_image_array_1 UniformConstant
 Texture2DArray<T> tTex;
 
+// Just to prevent DCE.
```

Comment:
You should be able to remove all of the additions to this file since DCE isn't run on these tests

Reply:
Right, I thought this was using lit so no `-fcgl`. Reverting.